samples/boards/nrf/ieee802154/802154_rpmsg/boards/nrf5340dk_nrf5340_cpunet.overlay

```diff
@@ -0,0 +1,35 @@
+/* Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+&uart0 {
+	status = "disabled";
+};
```

Comment:
Are such changes actually needed? Since the corresponding driver is not enabled (`CONFIG_SERIAL=n`), the above should not matter.

Reply:
Actually nRF53 does not need it. I attached it since similar configuration is necessary for the nRF52840.
I will remove this file.